### PR TITLE
Layer5 website updates

### PR DESCRIPTION
**Pull request by Shivam-AfA**
**Description**

This PR:
- Fixes the crash on news page in dark mode.
- Changes CTA from docker extension to MeshMap.
- Adds Meshery Cloud repo in the repository overview page.
- Changed review slider settings.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit f6c32ebe0785ff80d3c5ce26a59ed97d3d3ffe2b at: https://6458e6438f69eb26a6488466--layer5.netlify.app

**Comment by Shivam-AfA** [comment]
Sure @Chadha93. You mean Meshery Cloud, right? And what should be the description for the repo? // @leecalcote

**Comment by leecalcote** [comment]
> Sure @Chadha93. You mean Meshery Cloud, right? And what should be the description for the repo? // @leecalcote

“Remote Provider for Meshery”

**Review by thoughtlessmind — APPROVED** [review]

File: src/components/Card/index.js
Comment (randychilau) [inline src/components/Card/index.js:1]: @Shivam-AfA I noticed that there were some other frontmatter objects missing darkthumbnail after it was reported that the blog category page was crashing in darkmode, but was handling case by case. Your improvement on the conditional handles all cases (wish I thought of that), nice fix!

Comment (Shivam-AfA) [reply src/components/Card/index.js:1]: Thanks @randychilau 😀

File: src/sections/Community/Handbook/repo-data.js [hunk 1]
```diff
@@ -116,6 +116,15 @@ export const repo_data = [
         link: ["https://layer5.io/community/members/nikhil-ladha"],
         repository: "https://github.com/meshery/play",
       },
+      {
+        project: "Meshery Cloud",
+        image: meshery,
+        site: "https://meshery.layer5.io/",
+        language: "React.js",
+        maintainers_name: ["Vacant"],
```
Comment (Chadha93) [inline src/sections/Community/Handbook/repo-data.js:124]: Add maintainers @theBeginner86 @MUzairS15

**Comment by l5io** [comment]
🚀 Preview for commit e14e1bd639da25cfa01086858cdf276ecef58e65 at: https://64593a82063c6c0e5b93f341--layer5.netlify.app

File: src/components/Card/index.js
Comment (leecalcote) [reply src/components/Card/index.js:1]: 😎

File: src/sections/Community/Handbook/repo-data.js (hunk 1 above)
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:124]:
```suggestion
        maintainers_name: ["Pranav Singh","Uzair Shaikh"],
```

File: src/sections/Community/Handbook/repo-data.js [hunk 2]
```diff
@@ -116,6 +116,15 @@ export const repo_data = [
...
         repository: "https://github.com/meshery/play",
       },
+      {
+        project: "Meshery Cloud",
+        image: meshery,
+        site: "https://meshery.layer5.io/",
+        language: "React.js",
+        maintainers_name: ["Pranav Singh","Uzair Shaikh"],
+        link: ["#"],
+        repository: "https://github.com/meshery/meshery",
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:126]:
```suggestion
        repository: "https://github.com/layer5io/meshery-cloud",
```
Incorrect repo.

**Comment by l5io** [comment]
🚀 Preview for commit 33d44146f1dcf43503f256c9a37680f0a01b3da2 at: https://6459cb624a9fa417dedba00c--layer5.netlify.app

**Review by leecalcote — APPROVED** [review]
🚀

**Pull request merged**